packages/jade-compiler/lib/transpilers.js

```diff
@@ -440,8 +450,6 @@ _.extend(TemplateCompiler.prototype, {
       content = self._optimize(content[0]);
     else if (interposeEOL)
       content = self._interposeEOL(content);
-    else
-      content = content;
```

Comment:
Oh, I removed that because an eslint rule was making the tests fails, complaining about auto assignation. I didn't found it to cause any effect so I took it off.

I'll add some tests.